patchwork-events-world/src/main/java/net/minecraftforge/event/world/WorldEvent.java

```diff
@@ -159,35 +158,25 @@ public BlockPos getPos() {
 			return pos;
 		}
 
-		public List<SpawnEntry> getList() {
+		public List<SpawnSettings.SpawnEntry> getList() {
 			return list;
 		}
-
-		@Override
-		public boolean isCancelable() {
-			return true;
-		}
 	}
 
 	/**
-	 * Called by ServerWorld when it attempts to create a spawnpoint for a dimension.
+	 * Called by {@link ServerWorld} when it attempts to create a spawnpoint for a dimension.
 	 * Canceling the event will prevent the vanilla code from running.
 	 */
 	public static class CreateSpawnPosition extends WorldEvent {
-		private final LevelInfo settings;
+		private final ServerWorldProperties settings;
 
-		public CreateSpawnPosition(WorldAccess world, LevelInfo settings) {
+		public CreateSpawnPosition(WorldAccess world, ServerWorldProperties settings) {
 			super(world);
 			this.settings = settings;
 		}
 
-		public LevelInfo getSettings() {
+		public ServerWorldProperties getSettings() {
 			return settings;
 		}
-
-		@Override
-		public boolean isCancelable() {
-			return true;
-		}
```

Comment:
This event is still cancellable in forge, this should be readded.

Reply:
Don't know how I missed that.